src/handler.ts

```diff
@@ -9,56 +9,40 @@ import { MCRequest } from './utils/MCRequest';
 const handler = async (
   event: SQSEvent,
   _context: Context,
-  callback: Callback,
+  _callback: Callback,
 ) => {
   let { NODE_ENV, SERVICE, AWS_REGION, AWS_STAGE, SEND_TO_SMC } = process.env;
```

Comment:
I know this is pedantic and not a change you've made, but can this be a const?

Reply:
Yes, done and pushed.